### DISCUSSION TITLE
How can users see their own previoys chats?

### DISCUSSION THREAD

**Discussion by fabr2004** [chatwoot/chatwoot#9494] [Q&A]
Consider this case:
1. A customer/user have a first conversation started and finished.
2. Same customer/user access chat to start a new conversation, the second chat in this scenario.

Can that customer/user see its own previous chat?
Therefore, can that customer/user see its initial chat?

Please notice that this is not related to Chatwood agent.

**Comment by vishnu-narayanan** [comment]
The customer will be able to see previous messages as long as the conversation is not resolved.
